docs/content/docs/connectors/datastream/jdbc.md

```diff
@@ -32,8 +32,15 @@ To use it, add the following dependency to your project (along with your JDBC dr
 
 {{< artifact flink-connector-jdbc withScalaVersion >}}
 
-Note that the streaming connectors are currently __NOT__ part of the binary distribution. See how to link with them for cluster execution [here]({{< ref "docs/dev/datastream/project-configuration" >}}).
+A driver dependency is also required to connect to a specified database. Here are drivers currently supported:
+
+| Driver      |      Group Id      |      Artifact Id       |      JAR         |
+| :-----------| :------------------| :----------------------| :----------------|
+| MySQL       |       `mysql`      | `mysql-connector-java` | [Download](https://repo.maven.apache.org/maven2/mysql/mysql-connector-java/) |
+| PostgreSQL  |  `org.postgresql`  |      `postgresql`      | [Download](https://jdbc.postgresql.org/download.html) |
```

Comment:
Are you sure we can link to this page? (PostgreSQL has its own license)

Reply:
It is using the same link as 
https://ci.apache.org/projects/flink/flink-docs-master/docs/connectors/table/jdbc/

So my take is `yes`